tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -35,13 +34,100 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
 
 		By("Deploy OperatorGroup")
```

Comment:
Please re-write this By. You deploy operator only in case when the "if" condition below is true